modules/candidate_list/README.md

```diff
@@ -0,0 +1,51 @@
+# Candidate List
+
+## Purpose
+
+The candidate list is intended to provide a menu from which users
+can access specific candidates while browsing LORIS data.
+
+## Intended Users
+
+The candidate list is primarily used by data entry staff to use in
+order to access the `timepoint_list` module (and `instrument_list`
+from there) in order to access the candidate and perform data entry.
+
+## Scope
+
+The `candidate_list` module provides a list of candidates, as well
+as linking to other relevant LORIS modules where applicable.
+
+## Permissions
+
+Accessing the `candidate_list` module requires either the `data_entry`
+permission and at least one study site, or the `access_all_profiles`
+permission.
+
+Users with the `access_all_profiles` permission can see every
+candidate in LORIS, while users with only the `data_entry` permission
+can only see candidates at study sites where they are affiliated.
+
+## Configurations
+
+The `useProjects` configuration affects whether the table has a
+"Project" column for the candidate, and whether there is a filter
+dropdown for that column.
+
+The `useEDC` configuration variable has a similar function for the
+"EDC" column.
```

Comment:
Because nobody knows what EDC is, I'd add: 
"And is used for studies where prenatal visits are registered (EDC = estimated date of confinement or expected birthdate)"

Reply:
I agree, it would be good to at least define what is EDC as not everyone knows about this.